icons/bootstrap/bootstrap-original.svg

```diff
@@ -0,0 +1,14 @@
+</defs>
+<g id="surface2">
+<path style=" stroke:none;fill-rule:nonzero;fill:url(#linear0);" d="M 14.121094 16.726562 C 13.878906 8.023438 19.28125 0 26.585938 0 L 101.421875 0 C 108.726562 0 114.132812 8.023438 113.890625 16.726562 C 113.65625 25.089844 113.960938 35.917969 116.132812 44.746094 C 118.308594 53.601562 121.984375 59.199219 128 59.921875 L 128 68.078125 C 121.984375 68.800781 118.308594 74.398438 116.132812 83.253906 C 113.960938 92.082031 113.65625 102.910156 113.890625 111.273438 C 114.132812 119.976562 108.726562 128 101.421875 128 L 26.585938 128 C 19.28125 128 13.878906 119.976562 14.121094 111.273438 C 14.351562 102.910156 14.050781 92.082031 11.878906 83.253906 C 9.703125 74.398438 6.015625 68.800781 0 68.078125 L 0 59.921875 C 6.015625 59.199219 9.703125 53.601562 11.878906 44.746094 C 14.050781 35.917969 14.351562 25.089844 14.121094 16.726562 Z M 14.121094 16.726562 "/>
+<use xlink:href="#image5" transform="matrix(0.5,0,0,0.627451,0,0)"/>
+</g>
+</svg>
```

Comment:
This suggestion optimizes the SVG. But please, **avoid using direct images in SVGs**, as we only allow pure SVG format.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128" height="128" viewBox="0 0 128 128"><defs><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="76.079" y1="10.798" x2="523.48" y2="365.945" gradientTransform="scale(.25 .31372)"><stop offset="0" style="stop-color:#9013fe;stop-opacity:1"/><stop offset="1" style="stop-color:#6610f2;stop-opacity:1"/></linearGradient><image id="b" width="256" height="204" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQAAAADMCAYAAACcL30dAAAABmJLR0QA/wD/AP+gvaeTAAAbJElEQVR4nO2de3Qc1X3Hv3dmdyXZsiXLsmUbAg7YBszT4KaFkCbFEEJogPA6pgSSQ05I0lMTalIoEAwJpbE5jksIpCGUQgkhxSEQmhhIDiGBug6EulCexRi/H9iW5IckW9LO3Ns/tKO9M3tnZne1O6t1vp9zxpq5j5m74/n+fr97Z+YOQAghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQshBiah1A/6IECHr9YSK2SZ1Rr1eiPWEgP8824G8esAkdNeQT4NQZ9TLBViP6MK3tfXg33ohzPt7f92QdDKKqbeLsF7wxG9r62HnerT/HxQjZAW/8GkM6oTRfvHVIz7xb9269diOjo4bAUApJQBASpnceS9SegXu3ZV7YCHrZt3dWTe7Y/fu3e9s27Zt64MPPrj9gQceGEShyPVdBY0BDcEohQag8ljIe35r7969F40fP/6RqhxJGVdjEv15oUX0jKGrxAHUPkD1uK7bpZTa5bhy3datm5+44oor/vDyyy9nDbtT2kJDQA56BIYMQAZAA4DG7u7uz6lqIPOLDC5uxOLkF9e0ZMMWqdxBqZzgkpV92az7bnYw++vdnbsXXHfdde0AmgA0aktD7pykc+eHjmeUwP+IyuJ5/+GLvLOz85KJEyc+XLEjRHn9OL8a5fXjQgFVRLQg4AJygyvdt95d8+7iE0444bXAnk0RAaOBGmLVugEHEabR/VQ2m63MOdakYlRNXLivIupFKVsBKkz8wboKNpR1pC3S580+5rhfDPZnn163bt3H4D83pgFSUiNoAKpD3Mh/8QREVpKAo4Rv3JmWoVS48CPr5gyGi4kC9ic+dOjhy/sPDDyxfPnyqcifEwv+sRIagRpBA1BZ9Pv9ACBc1y3/4g4IXwXzyh3ki7IIOdVHCj8kU+Wq+5Ci1RbpT11w/oXPd+3q+gqAVC7HMwSp3MJrsQbwpFcPG4Ao65ZfLcP9sCIxBqdA+IG6QlrTW1rabhvoH3hwwYIF4+CPBrwuAa/HhOEJrxymJ/1KE3+9hvsxbRredNV4G+mLvrN02S/vvvvuicifH90IsDuQIDQAlaXuwv1Y4ZcS7hvaFEQqCEj7I1/9ylefeeqppw5BoRFIgUYgMWgAqsOw94/tAkSF+8UIv+RwP1+pHOEDRQjfkK+8et7hHfu4cz716eU33XRTK8x3CWgEEoAGoFYUE+7H1C0v3Efxt/WC2SWE+wXNMeVJe86iW2577JRTTsmAtwhrAg1ALSg33A9EC1H5BRnV6udH1PV5fRMSsFXqtBd/9+I/IB/6ewODdkgtUkFoAJJkpOF+WLEqhftFCT8u3I+qO1TOzqQbL3nzzTfnIi96dgUSIhVfhIyYqFA/NNGfN+gMvus42W2mfEtYUXuIxxKNlmU1CIgMBBotJZqEZbcKoNloAKKihZh8U550RcesmUctaWtrO7O7u1vkSnldAddci1QCGoAEKbmfr63u27v395M7Jt+KvN91ijlEERR42FmzZtlLly49as6cOR9vaZkwJ2Vb09LpzGFQ1iTIiOaW2U0AAKHsY996653Lp07teARAFn7vL0AjUBVoABKgJOFr+XqxnJeXKHytNhhs61N1xRHsZwsAWLNmjTrvvPPeAvC2l7Fs2bJDP/dXV3xlfMv4022RngE1VDdW+CH5KrAiHYyZ2NZ2dVtb24+7u7tTGDJwjAKqDMcAkqbM0X1hCc/re9kSQ54ym0v31mUJSzawDAb+urlyauHChVsmT5n0jY+eftond3XvvEcKZ2NRXl+FJBnyhErNfOWVV86E+WEqjgVUARqApChidL/I23q6+CX8UUAp4tcXZViCBsIFIFevXj04bdqUxfd+/56LHTnwihCGiCNicNDwe4aRjhp7yLRDv4T8A0FAPgogVYAGIAmKHN035uUycl0AT5jBSMBF3lsHvXoxi2tYJAqNgZPLUwsXLtzysT8//eJ+Z//z0I1A2F2BwO8JK2TBPuqSSy5pQP1PoloX0ADUiiivHy4UUzfAMwhOIK+URa+vdymCxkBqeXL16tWDc+acdFXW6X9puHWmn1HkLUEAELCnLV68+M9hDv9pBCoMDUDSlBbuDxMyBmASfaW6AN7xgmML+rHl2rVrnZtvuelLSjhrjT+jhDECBcB1VEPH5KkXwR/6sxtQJWgAkqTIcL8gQ6lg1eDov8kolLKEDR7q+/fKuNqxsgDkXXfdtXv7jm0/FBb6Swn3g0keKTs1Lbca9PiMACoMDUBSlBzu50MFBUD4H/ZR8E/LHYwGyvH6Xn1XWw8aAj1dryfnzp37iCuza0oJ9wt+fm5DWEJ/VRjgE4FVgwYgAUoN94eFr0L1Egz53UBaKc0y7c8TuMkQKPiNgATgdHZ2yn09PS+KMJkawv2g8POZovXqq69ugln4NAQVhAagFsSF+4XCD9YO3norRfjFEjfW4G17ZeUzv1rxIwjVHdcyFbmhIBQazzjjjImB9lD4VYAGIEmKDPdL3GMpT/6VS1D0pmjB/fznP79RKucDXy3DTgo3AoWVyHR0dDSjcPCPA4EVhgYgKUoP9/P14q1Cpb1/2DGCnj8YBSipZFdJ4b6pMNDY0tLSXMnGEzN8FyAJQsP9GOUmIevSUMi/mBNcBwAlpdxni3zhgtrmDf+mEI7jONnKNJlEwQggcYoI96vRo68cXrfDa6FvXUo5CEQ7+fDwILeq1OCePXv6KthmEgIjgEQpQvijH8/z69vDEYCwLOkL9wuqmjf1HCnU/meffXbHiFtKYqEBSISRhfuRs/LUDmOrxNCHQIsO9002Qii1d9myZT0jbiGJhQYgCcr0+qNU+JFYwhobGe6bc3wJrnS6Kt8yYoIGoFbEiLsexQ8AlmW3Dd2YLM3rDyOAgWx2nZbjGtZJhaABqAUHh9cveEPvnnvumSSENVmFqD1uSAAA7BT6Nq5f99OKtZJEQgOQJAeP8As+ggoAl1566YWQYgqAaOGHJgIS7qZzzz3391qpYMn6OVN1AG8DJkHMbb1ypt6uIcFHcoeNwbjmlvOUhIj1+hG/J+sMvrp161Y3UFJhNJ6JgwAagBpS7pd2aoj+FV/f133fePWNj9rKPtprc9wDfwUoQKTUjhXP/PJOrQapMuwC1IDYcL/0Sz+JabP1D3d6UYANQBx99NH2zKNmfVNmxYRSwn09TwnAlQOvzp8/f5OWwwHAKsMIIGHK/dJOCEnMlON5+ZS2jty2BUD84aVXvi3c1Aklhfv5ByKhAFgptf0XK/7jFvjD/WDoz6igwjACSIhyvq03nOXP90TvBNJi9lQSwRF+PewH8uK3Pti2868b0k0XSQdpf6MjUL4/sGzh9g30PHPZZZe9r5UwvfJMKgwNQAKU+2FNQ55pvnzPI+uCKUYsYe/X66LXj1Ug/u7OPV8f09T8ZemItqKOrEKK2M7bl1xw4SLkvX1wOjJ+GKRKsAtQKyLCfdOHNZWSwdtuukC9EF0P0+MWXdgpw2IqmwJgLV68uHV/X//DzU3jv6Yc0R71e/Q8UxEro7b856oXrnn++ef3a6Vdf02Kv1owAkia0sL9IPqHM0VgG/B76lIouKcf2LYBWPPmzUs/vvzxa8aOHT9fDmKWGzN1EYBwrw/ATqF7+65td5111lmvI9r7kypBA5AUIxG+AqRUngf23pP3vH5QJJWYOkuPMKwnnnjikLPOPPvaTDrzUeVYR7kDSI1E+FCAncK+vb1d90+fftjD8M8v6P2W4MSlpArQANSQIjy+aQDQ+3AmMGQEvAjAK1mOx9TvJIglS5a0zJ8//88mTmg/N53OzBbKOlw66Bh60z+mzVHCz2XYaXTv7e26f/LUyd/RfotC4ZRj7PtXGRqAGqAKVqIKDdHU1Dh748YtVwGAZVkqlbKlZeWnCtfXwz7hnUeITMZuVgJ22k5NEMJqF0KMtS17HIA2AWuCkqLDdVVqWPQltDfqWQCrQW3Zun3L3UfMmP4QCqcx58BfwtAAJEwx4b6JFBpOnTpp2qmlSMJ4X17f8LrwOelJX7E4RRfuN0r4whJSpLNv//bF568955xz3oD5K0daaxj6JwENQEKUEe77k6K8elQ9Y75Z3FECjjpuVLgPAYi03L6/v/dX5//lZ25duXLlAMwfN4WWRu+fELwNmACm23r+AoX5w0nF1A3bVUHdQEKU+Iu4rWesl0sUGBJ+VvU9e9+/3PvZie0Tbly5cuUgCkN9Fdim+BOEEUAtKdlzh+cV7fEDm3FFw45rKiYEIGy121GDm/r6el/9p+8uu2fx4sU74R/oC37FKOj9i2kFqRA0ALUg5PIuZ3CwICnKElQx3Leb3E3btm9/8sknf/bkwoULN+aS9f48AuvBbYq/BvBzS5XDuy/vPTmXBmBt2LDhwkOmHPaA8o+w+ShK+Ib8RLx+Mf18AFYGBxTkHqXkPkc6uw4c2L9xzXv/t+LKK69ctX79+oFcae+Lwrr3Dw7+6R8bIVWGBqByRBsAw935eg73i5GolRYDEtktrnTWbti0Yfm8eX+xYufOnd5nxvXR/+CAIO8AJAQHAatNyJBW+EBdoFDR4jcM8CljjqGu+bih9WLqehOduIOqQQ2mjrScxrNnTj/mno3rNj/X1bX7G1ddddUYFD627G0HpxsjVYQGIGGqN7pvzitndD+0SDHCD8l2B1ST7E8d12SNv/b7d9/32+7OPX936qmn6i8wATQCiUMDkBAlCT9Mz0bhm9UeV9R43LAiMUYj8p2gYBNdCLffmtVoj7vht8+98Js333xzLvwzDfnePIxoMakAPMEJUPTofpyAiyhclvBHEu7H7DcM6cB29tsnzTj8mJ/s7tr3zfb2dt0IeOMo3kQkpErw5CZBGeF+eN0Sw/2YNlU83I/5rcGJUJ1+dDTYY7+6ecOWxxcsWNAM/0Qn7A5UGT4HUEuKHuADkHK3Kel7NQdCCF+polWiRQtiaEcZoYStFNKWEo1KIAOFtJKAkv56UXYhTvhhyCwahZM+a+mdy34+ZcqUi2+++eY9ub1ZGLoz4L0BGXl4Ujo0ALUgSvgFCQoQwN69u3/dMXXyrcj7WCdYzbQrAwV24vTTT8+ccsopTSeffHJLa2vr2HHjxjVNmDChddKkjtnNY8bOtCy73bZTHRbsdiHtSdJR/sgx5qjFTImmFITTZ5329b+9/uebN2/+zA9+8IMeaDZK+0sjUEEYWlUO83MA72+4cNrU3INAhks30ut7CQLY17f7wclT2hfBf2NRaYtH1HwAphmETTMAGfNuuOGGiV/64pfPmTR58pmZdGYGsvbh0gl3IuXMhSgsKKsx+5vjTzruorVr1+oPDfEZgSrAMYCkMHj9Ukb3LUt4Xl8XRDa3ONq6jFiyhmUwJD2LIcG5Xv0lS5Z0zph1xCMtrc1fmHfWJ+Zt79x4I9ID/2OllDef31BzVXniVwCkCyEH0x9/dfX/LoH/7oAFjgdUHBqAJBjJbb1CsejiD747HyV+fVGGeqZ9hBkFuWrVqv4jZhzxyNiWMeeu2fjONWjIvi0syHK/e5B/Y1JBZlU6hYaL3nvvvU+gcHZigEagYtAAJMhIbusJYekCNT0+63nqMG9u8uxhS5Sh0PcjAagTTzx+xVlnn/Hpftnz76kM9hh/eITXz/+TL+T0Y9KHpk2/4+yzz84gP1bFuwIVhgYgAeK9vnkzxFkGxe8ZBAeFc+qFLY5hMXUnTIbBZAzkqlWr+tvaW6/bsmvTt+0GtcP8WwPnxO/6Cwo4+8Xsx37y2I0ofEiIRqBC0AAkRQnhvqloyBiASfTldAH0Q5oMhW4YgsbA1x2ZMePDD2/ZvvFOK606Q09DlPCRT1Yu7MZ08wXXX3/9OPi7AvpfMgJoABKnBOGjIMEkUn07GA2ELWGDhS7iDYW3j+Co/HA0MPOoIx/tG9jzuJXC8HMLWhff9CtDT4DTjyP//oZvXIvCj5kwCqgANACJEj7AZxS+lpgbA9Bz9Zl0gtFAMV4/aBQ8z66nRe1Hr+c9LuS1Qx5z7NHfQsZ5ffi3xYT7YcnShRjT0Hj2hAkTgl9D0v+SMqEBSIRwtcf1DEJ2FAz5g9NsFVs3qusQFHnQKOiGwEtD7q/T2dkpX3t19T9aGewuJtw3noLcisraH37xxRc/gsKxAIBGYETQACRJTD8/QvjBkqYv5xZXu3hMxiHYBfBEr3cr4OWd9rHTVsFy3o3cuykpkJcdUOM+fNiRV6Hwm4jV/jT6QQ8NQBIUE+4XWdeQm+S383Sj4EUd+mRnQaOgOrt3PW2ltHcWooSPwjwvzRKpGTB7e0YAI4AGIEFK9vrK9ydu10kSHHfQRe8bb3jo4X99wkqrHVH9/OGViM6LgDXltttua4G5G0DKhAYgIUoWvqpOXF9hguMIgP9dBWfRokW7lHC3h1Us2ghmxZTLL798DhgFVBQagKQ5OISv43UHgg8JDRsG6cpuvXBkuG8sCLiOSrW1tp8MjgNUFBqApChmfD6syOi3BirwV48CXEc5e/XMYs+FniRdIJNpOKoyzSUeNABJUGQ/3zg4OPrFD4TfmRj6q2R/KeG+vqkbDQGYZhMmI4ATgtSK+vb4YRglLpXqtU0Zeq2wTd+GyARq6s8C1O9ZqyE0AEkTN7Jfv5dxsOXDxsAW1hjja8JRwjfvMV1+84gJGoAkOTi9fizCspsKEov2+vqORNaUWmazCGgAkkEb3TfmxdStd9sgIFrDRv6LEn4OKV3dADDsrwA0AAkQOvYVMyBWZ1e3aW5BAUBYwp7glhrumw4gMBhfipQCDUAtqMzTf6ORghH622+/vVXAnqKnleL1h3csAMfNdmtJdXqKRhc0AEly8Ib7Yd4fX/jCFz8jBzAFKE/4HlZKYHdv738bsur0lI0O+BxAUsSF+1HiH/2XeHCCjuH1iS1tF7sOLN8YQDm/x1Y969evfX2kDSV+GAEkQcRTb5Fa8DJH9zh38Mu+w8tzzz13jJKp/NN7IzBkdgpdd9555/pAcvAJRFIijABqRZzHr49LWv+Sr+79LQA47U9Pv8M5gMmV+D0OstuefvrpftTT2akDGAEkTXVC/VrcEguK3/dl3w82f3Cl22//yfCLwiM5kAAcJ/vOyPdEgjACSIoR9PMjPrZRi7fhPJGnUCj+FACx+uXVJ48f136dHBTNlTignUHf++vXLM9t6u8dmN5BICXACCAhRiR8/xiAJ3onkBazt7IJfivQMzgW/P3/FADrpZUvHT979ok/HNyHwyrWgoz7/ty5c1/JbZm6AOwSlAkNQJXR34v1EXPJhnj94IAbkI/idE9YiiDChhj19KjPc3lRgLVx45bzJ7dOub2S4hcWMNB/YCXywtcH/ij8EcIuQC2I8fom8SslwzyxKSS3SliCIby32Np6VHlr6dKlbft7+h+Y2Dzlu4O9FfT8AFJN2PLjx370Pe80GIrQCIwARgBJUl4/X8cb7NNF6G0Dfg9dDlGfCtfTbADW/fffP/HSS+Zfn7GaPpntxRGqAgN+vgNZwKA88F8LFizYkUti/7/C0AAkQUywWsyntKVSnsf1XojxvH5QCJV+asAXbdx0003NX1vwtQvGN7dcoNzUbKcfhw5WSYapJrnxO9+761b4w39TV4CUCQ1ADSlG+AE8I+ANAFrIRwBejZHIcfiOQnt7u7jvvvsmHX/88Ue0tbZ/pKlxzBwB6xBI6xB3EIcO7qtu99HOiMHe/n1P3XLLLTtzSUHvr/8lZUIDUANiw31D/tixY+ft7xk4CQCEEMoSouiLfzgyj37sUCoXlmWLBiiklBTNCiKjXLTJrMo4PcUerQIIQDQMvn78zGPvQN7bexONAnljQAMwQmgAEqYMrw+lgGyPPR3A9PBCEUlFGRzh3/TWaiCxzFhsXvHrFX+zY8cOr7tD718laAASohzhD2epiEJxySUcdyRv61WK1BjVuW3npsUXX3zxeyj0/t46B/8qBA1AApTl9f3/FF23HOEXJNVI/HYD9u3t6/znI2ce+VPkRe99gUhvGZ8BqBA0ALUiVvhxhUKSShxfGA3CB4BUE/b09Hc/NO3Qqd9D/k6Hgl/87PtXGBqApKlwuF+0gEep8AEgNUZ1de3ddddh06fdh7zg9S8SAxR/VeCTgEkSoWuV/yekQMTuogJiQ95oEb8QQGacXLt+03sLNfF7Yb8e+vs+NVaTxh6k0AAkQZSuh/NK8/rKtxJx3LBm1FhKqTT6rebsC3fc+a3zjjtx9jOIFz+9fxVgF6BGjCjcr9N+PgBYNlRqjFy7q3vnv80+4uj7e3t7FfJ9/qD42e+vMowAqsOwf7Usq0B/Iw73Y48asqtahvs2kBorNw5YPQ9dduWlnzxs+qH39/b2OogWv3fLj6F/lWAEUFlCL9Kq3tYz5I8W4dsNIou0s+5Af8/zt91861333nvvHuRH9/V+PcVfA2gAKoc+Wu09mx/Q/R9HuG+nhRQZuU3CWbej64MnPvvZ83/2+uuvDyIfzntPJ+vC91ob7PNT/FWEBqB6uAAs27ZUqPjj7EG5t/Xi6lYSAdgpAZFWuyDcDxyZ3bKru+t3Ny+68fFHH320D/5+vC58k9cP9vkp/ipDA1BZXPjfyVeWZZsv4pH086OSKiQZkRsdErlfIywBYale2OgTQh6QUvZI6exy4O7o7O7+w4pnfvHCNddcsx3RQtaF7xrSg8aAVBkagOowfGFv3rzhjZZjx9+nbCmkVMKb2UdKNapn+5dKqsEBZ4+E62Szg519+3p2rN+0fvNrr73WuWjRoq5csaCAAbP31o2CE8hjyF9DRvVFWIdYyM+Lr8+Z502nBeQjBBsjn8EnKYKi1MXubZvKmKIBUz7FXyMYAVQW72LWp+gyzdbjhcH1aADCRBqMBsLu31P4o4h6uPjqDS8K8Cbn9AjOoV+v517F/I0Svbce7P+TGlGvF+FoxhTiB6fStg1po5W48N9URk/TvX0wndSY0X7x1StRRsBUdjRTrFCjIgJvm6IfZYz2i6+e0UVv8vj1eO7DxgL0dXr6OqIeL8J6I+j97UBePWAScXBarmIGCskoo14uwIOB4Jd96pGovj4hhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGE1AP/D9zQkSwrq6iWAAAAAElFTkSuQmCC"/></defs><path style="stroke:none;fill-rule:nonzero;fill:url(#a)" d="M14.121 16.727C13.88 8.023 19.281 0 26.586 0h74.836c7.305 0 12.71 8.023 12.469 16.727-.235 8.363.07 19.191 2.242 28.02 2.176 8.855 5.851 14.452 11.867 15.175v8.156c-6.016.723-9.691 6.32-11.867 15.176-2.172 8.828-2.477 19.656-2.242 28.02.242 8.703-5.164 16.726-12.47 16.726H26.587c-7.305 0-12.707-8.023-12.465-16.727.23-8.363-.07-19.191-2.242-28.02C9.703 74.399 6.016 68.802 0 68.079v-8.156c6.016-.723 9.703-6.32 11.879-15.176 2.172-8.828 2.473-19.656 2.242-28.02zm0 0"/><use xlink:href="#b" transform="scale(.5 .62745)"/></svg>

```